CHANGES.rst

```diff
@@ -15,6 +15,7 @@ Unreleased
 - Added pagination to ``SimpleHistoryAdmin`` (gh-1277)
 - Fixed issue with history button not working when viewing historical entries in the
   admin (gh-527)
+- Updated test matrix to test against Django 5.2 explicitly.
```

Comment:
Would you like to change our convention with how we write release notes for new Django versions? 🙂 (Many of the previous versions we've written "added support" for, have actually just been added to the test matrix like in this PR 😅) If so:
```suggestion
- Updated test matrix to test against Django 5.2 explicitly (gh-1441)
```
Otherwise:
```suggestion
- Added support for Django 5.2 (gh-1441)
```